pkg/prowgen/prowgen_konflux.go

```diff
@@ -75,16 +81,11 @@ func GenerateKonflux(ctx context.Context, openshiftRelease Repository, configs [
 						}
 						// For non-existent branches we keep going and use downstreamVersion for versionLabel.
 					} else {
-						soProjectYamlPath := filepath.Join(soRepo.RepositoryDirectory(),
-							"olm-catalog", "serverless-operator", "project.yaml")
-						soMetadata, err := project.ReadMetadataFile(soProjectYamlPath)
-						if err != nil {
-							return err
-						}
 						versionLabel = soMetadata.Project.Version
 					}
 					log.Println("Version label:", versionLabel)
 					buildArgs = append(buildArgs, fmt.Sprintf("VERSION=%s", versionLabel))
+					buildArgs = append(buildArgs, fmt.Sprintf("MUSTGATHER=brew.registry.redhat.io/rh-osbs/openshift-ose-must-gather:v%s", soMetadata.Requirements.OcpVersion.Min))
```

Comment:
as we need the mustgather arg only for SO builds, I think we can remove it here (have it in `GenerateKonfluxServerlessOperator`)